src/query.ts

```diff
@@ -278,6 +290,24 @@ export class PreparedQuery<R extends Row = Row> {
     return (row as unknown) as R;
   }
 
+  private makeRowObject(row: Row): O {
+    if (this._rowKeys == null) {
+      const rowCount = this._wasm.column_count(this._stmt);
```

Comment:
sorry, I know this pr is already closed but are we always making two database calls when we grab row objects? Is there not a way to return all this information in a single call?

Reply:
You have to do this at least once to collect the rows names, but the names are memoized for subsequent calls.